routes/search.js

```diff
@@ -1,11 +1,13 @@
 const express = require('express');
 const router = express.Router();
+const { requireParameters } = require('../middlewares');
 
 const logger = require('../modules/logger');
 
 const searchController = require('../controllers/search');
 
-router.get('/sellers', (req, res) => {
+const paramMiddle = requireParameters(['longitude', 'latitude', 'searchText']);
```

Comment:
regarding `paramMiddle` varaible - on other places we have named variable as `googleParams` (routes/seller) and `signupParams`. I think we can be consistent. my suggestion - `requiredParamsMiddleware`